flink-runtime/src/main/java/org/apache/flink/streaming/api/transformations/OneInputTransformation.java

```diff
@@ -192,4 +192,9 @@ public boolean isOutputOnlyAfterEndOfStream() {
     public boolean isInternalSorterSupported() {
         return operatorFactory.getOperatorAttributes().isInternalSorterSupported();
     }
+
+    @Override
+    public void enableAsyncState() {
+        // nothing to do.
+    }
```

Comment:
Here we should apply the change like [this PR](https://github.com/apache/flink/pull/26000/files#diff-a3035df99b00d9ab9a00e5050b2377d6cbb9ab7db0766ec9d88dc504a2643cfeR189-R198). Or after that is merged, rebase on that.